### DISCUSSION TITLE
Why multiple columns for future step predictions?

### DISCUSSION THREAD

**Discussion by aegek** [ourownstory/neural_prophet#214] [Q&A - get help using NeuralProphet]
So I want to make a 48 hours prediction from a given point but code returns me a table with gazillion columns (yhat1-yhat2...-yhat48). I'm collecting them under a single column manually but I wonder if there is a way for it that I miss.

**Comment by ourownstory** [comment]
Hi @aegek,
The format is such as to work with the plotting functions, and also when you do a prediction at multiple points in the past.

However, what you describe is a legit use case, and a simple helper function could provide that.
Would you like to share your code by making a pull request to add it as a function in `utils.py`?
